toolkit/tools/imager/imager.go

```diff
@@ -734,5 +734,11 @@ func configureDiskBootloader(systemConfig configuration.SystemConfig, installChr
 		return
 	}
 
+	err = installutils.CallGrubMkconfig(installChroot)
```

Comment:
If we are doing this, do we even need the old grub template?

Reply:
(The one from `InstallGrubCfg()`)

Reply:
I suppose we should clean up the entire legacy flow in this PR.. will make that change.